src/shared/components/Partner/component.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
I guess these changes are taken from https://github.com/MecenatOrg/frontend/pull/7 PR

please remove them, as they are not related to the current PR.